crates/editor/src/code_context_menus.rs

```diff
@@ -859,8 +891,10 @@ impl CodeActionsMenu {
             },
         )
         .elevation_1(cx)
-        .p_1()
-        .max_h(max_height)
+        // x padding is the same as y.
```

Comment:
We could use a single `.p` here:

```suggestion
        .p(Self::y_padding() / 2.)
```

Reply:
Ah true, thanks!!  Ended up replacing this code with use of `ListItem` + `Popover` in #22112